package javaMid10.p468;

public class ExceptionEx5 {

	public static void main(String[] args) {
		try {
		      A a = new A();
		      a.a1();
		    } catch (ArithmeticException | ArrayIndexOutOfBoundsException e ) {
		    	if( e instanceof ArithmeticException ) {
		    	    System.out.println("수학적으로 틀림" + e.getMessage() );
		    	} else if (e instanceof ArrayIndexOutOfBoundsException) {
		    		System.out.println("배열의 인덱스 범위가 틀림" + e.getMessage() );
		    	}
		    }
         
	}

}
